pkg/snapshot/snapshotter/snapshotter_test.go

```diff
@@ -306,7 +339,7 @@ var _ = Describe("Snapshotter", func() {
 								etcdUsername,
 								etcdPassword)
 							snapshotterConfig, err := NewSnapshotterConfig(
-								schedule,
+								fmt.Sprintf("59 %d * * *", currentHour+1), // This make sure that full snapshot timer doesn't trigger full snapshot.
```

Comment:
```suggestion
								fmt.Sprintf("59 %d * * *", (currentHour+1)%24), // This make sure that full snapshot timer doesn't trigger full snapshot.
```

To handle case when `currentHour` is 23.